README.md

```diff
@@ -15,17 +15,20 @@ ZenML projects using `zenml init --template`
     - Digit: `sklearn_digits_loader_step`
     - Iris: `sklearn_iris_loader_step`
     - Diabetes: `sklearn_diabetes_loader_step`
-    - Breast Cancer Wisconsin: `sklearn_breast_cancer_loader_step` 
+    - Breast Cancer Wisconsin: `sklearn_breast_cancer_loader_step`
+    - Wine: `sklearn_wine_loader_step`
```

Comment:
OpenML still missing here

Reply:
Also, note to self, update the config file...